addons/jam/jam_ws/CfgMagazineWells.hpp

```diff
@@ -0,0 +1,151 @@
+            "20Rnd_12Gauge_AA40_Pellets_Snake_lxWS",
+            "20Rnd_12Gauge_AA40_Pellets_Tan_lxWS",
+            "20Rnd_12Gauge_AA40_Slug_lxWS",
+            "20Rnd_12Gauge_AA40_Slug_Snake_lxWS",
+            "20Rnd_12Gauge_AA40_Slug_Tan_lxWS",
+            "20Rnd_12Gauge_AA40_Slug_Tan_lxWS",
+            "20Rnd_12Gauge_AA40_Smoke_lxWS",
+            "20Rnd_12Gauge_AA40_Smoke_Snake_lxWS",
+            "20Rnd_12Gauge_AA40_Smoke_Tan_lxWS"
+        };
+    };
+
+    class CBA_556x45_GALIL {
+        WS_mags[] = {
+            "35Rnd_556x45_Velko_lxWS",
+            "35Rnd_556x45_Velko_reload_tracer_green_lxWS",
+            "35Rnd_556x45_Velko_reload_tracer_red_lxWS",
+            "35Rnd_556x45_Velko_reload_tracer_yellow_lxWS",
+            "35Rnd_556x45_Velko_snake_lxWS",
+            "35Rnd_556x45_Velko_snake_reload_tracer_green_lxWS",
+            "35Rnd_556x45_Velko_snake_reload_tracer_red_lxWS",
+            "35Rnd_556x45_Velko_snake_reload_tracer_yellow_lxWS",
+            "35Rnd_556x45_Velko_snake_tracer_green_lxWS",
+            "35Rnd_556x45_Velko_snake_tracer_red_lxWS",
+            "35Rnd_556x45_Velko_snake_tracer_yellow_lxWS",
+            "35Rnd_556x45_Velko_tracer_green_lxWS",
+            "35Rnd_556x45_Velko_tracer_red_lxWS",
+            "35Rnd_556x45_Velko_tracer_yellow_lxWS",
+            "50Rnd_556x45_Velko_lxWS",
+            "50Rnd_556x45_Velko_reload_tracer_green_lxWS",
+            "50Rnd_556x45_Velko_reload_tracer_red_lxWS",
+            "50Rnd_556x45_Velko_reload_tracer_yellow_lxWS",
+            "50Rnd_556x45_Velko_snake_lxWS",
+            "50Rnd_556x45_Velko_snake_reload_tracer_green_lxWS",
+            "50Rnd_556x45_Velko_snake_reload_tracer_red_lxWS",
+            "50Rnd_556x45_Velko_snake_reload_tracer_yellow_lxWS",
+            "50Rnd_556x45_Velko_snake_tracer_green_lxWS",
+            "50Rnd_556x45_Velko_snake_tracer_red_lxWS",
+            "50Rnd_556x45_Velko_snake_tracer_yellow_lxWS",
+            "50Rnd_556x45_Velko_tracer_green_lxWS",
+            "50Rnd_556x45_Velko_tracer_red_lxWS",
+            "50Rnd_556x45_Velko_tracer_yellow_lxWS"
+        };
+    };
+
+    class CBA_556x45_STANAG_XL {
+        WS_mags[] = {
+            "75Rnd_556x45_Stanag_lxWS",
+            "75Rnd_556x45_Stanag_green_lxWS",
+            "75Rnd_556x45_Stanag_red_lxWS"
+        };
+    };
+
+    class CBA_762x39_AK {
+        WS_mags[] = {
+            "30Rnd_762x39_Mag_worn_lxWS"
+        };
+    };
+
+    class CBA_762x51_FAL {
+        WS_mags[] = {
+            "20Rnd_762x51_slr_lxWS",
+            "20Rnd_762x51_slr_desert_lxWS",
+            "20Rnd_762x51_slr_desert_reload_tracer_green_lxWS",
+            "20Rnd_762x51_slr_desert_tracer_green_lxWS",
+            "20Rnd_762x51_slr_reload_tracer_green_lxWS",
+            "20Rnd_762x51_slr_tracer_green_lxWS"
+        };
+    };
+
+    class CBA_762x51_FAL_L {
+        WS_mags[] = {
+            "30Rnd_762x51_slr_lxWS",
+            "30Rnd_762x51_slr_lxWS_desert",
```

Comment:
```suggestion
            "30Rnd_762x51_slr_desert_lxWS",
```

Reply:
Thanks, I found another such erroneous while updating to new structure but missed this one!